src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -135,7 +173,7 @@ private Instruction AddInstrumentationCode(MethodDefinition method, ILProcessor
 
             var pathInstr = Instruction.Create(OpCodes.Ldstr, _result.HitsFilePath);
             var markInstr = Instruction.Create(OpCodes.Ldstr, marker);
-            var callInstr = Instruction.Create(OpCodes.Call, processor.Body.Method.Module.ImportReference(typeof(CoverageTracker).GetMethod("MarkExecuted")));
+            var callInstr = Instruction.Create(OpCodes.Call, processor.Body.Method.Module.ImportReference(_markExecutedMethodLoader.Value));
```

Comment:
Any reason we can't just put `typeof(CoverageTracker).GetMethod(nameof(CoverageTracker.MarkExecuted))` here?